custom/src/test/java/com/splunk/opentelemetry/micrometer/GlobalTagsBuilderTest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.opentelemetry.micrometer;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import io.micrometer.core.instrument.Tag;
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.sdk.resources.Resource;
+import io.opentelemetry.semconv.resource.attributes.ResourceAttributes;
+import org.junit.jupiter.api.Test;
+
+class GlobalTagsBuilderTest {
+  @Test
+  void shouldBuildEmptyTagsList() {
+    // given
+    var resource = Resource.create(Attributes.empty());
+
+    // when
+    var tags = new GlobalTagsBuilder(resource).build();
+
+    // then
+    assertTrue(tags.isEmpty());
+  }
+
+  @Test
+  void shouldBuildGlobalTagsList() {
+    // given
+    var resource =
+        Resource.create(
+            Attributes.of(
+                AttributeKey.stringKey("environment"),
+                "prod",
+                ResourceAttributes.SERVICE_NAME,
+                "my-service",
+                ResourceAttributes.PROCESS_RUNTIME_NAME,
+                "OpenJDK Runtime Environment",
+                ResourceAttributes.PROCESS_PID,
+                12345L));
+
+    // when
+    var tags = new GlobalTagsBuilder(resource).build();
```

Comment:
java 11 in tests.. is there gradle magic that makes sure our code doesn't start using java 11 APIs that would break users on java 8?